gapic/grpc_service_config.json

```diff
@@ -3,34 +3,16 @@
       {
         "name": [
           {
-            "service": "google.cloud.apigee.registry.v1alpha1.Registry",
-            "method": "GetApi"
+            "service": "google.cloud.apigee.registry.v1.Registry"
           },
           {
-            "service": "google.cloud.apigee.registry.v1alpha1.Registry",
-            "method": "CreateApi"
-          },
-          {
-            "service": "google.cloud.apigee.registry.v1alpha1.Registry",
-            "method": "GetVersion"
-          },
-          {
-            "service": "google.cloud.apigee.registry.v1alpha1.Registry",
-            "method": "CreateVersion"
-          },
-          {
-            "service": "google.cloud.apigee.registry.v1alpha1.Registry",
-            "method": "GetSpec"
-          },
-          {
-            "service": "google.cloud.apigee.registry.v1alpha1.Registry",
-            "method": "CreateSpec"
+            "service": "google.cloud.apigee.registry.v1alpha1.Registry"
```

Comment:
I think we can drop this. In this branch (and following), the gapics are generated for the v1 API only.

Reply:
Makes sense. I'll leave it in for now as we still have the v1alpha1 protos, but it can be removed at any time. Even if we forget to remove it in the future it won't cause issues by having rules on a nonexistent service.